Makefile

```
@@ -337,6 +337,29 @@ ifeq ($(FAILBUILD),1)
 $(error cannot continue build)
 endif
 
+# Add bootsplash to BOOTSPLASH/logo.bmp
+define bootsplash_region_hook
+
+# Only run if a file is provided
+ifneq ($(CONFIG_BOOTSPLASH_REGION_LOGO_FILE),"")
+
+real-target: branding_bootsplash_region
```

```suggestion
files_added:: branding_bootsplash_region
```
Otherwise `BRANDING ...` can be printed after the final `Built ...` line.  This placement is also not ideal:
```
    CBFSPRINT  coreboot.rom

    BRANDING   Adding bootsplash (BOOTSPLASH/logo.bmp) from 3rdparty/dasharo-blobs/protectli/bootsplash.bmp
FMAP REGION: COREBOOT
Name                           Offset     Type           Size   Comp
```
but it's better and I think `files_added` is how such modifications must be done.

---

[Done](https://github.com/Dasharo/coreboot/pull/771/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R343)